source/backend/api/Repositories/Mayan/MayanDocumentRepository.cs

```diff
@@ -256,19 +256,14 @@ public async Task<ExternalResponse<DocumentDetailModel>> TryUploadDocumentAsync(
             _logger.LogDebug("Uploading document {documentType}...", documentType);
             string authenticationToken = await _authRepository.GetTokenAsync();
 
-            byte[] fileData;
             using Stream stream = file.OpenReadStream();
-            using var byteReader = new BinaryReader(stream);
```

Comment:
even though this wasn't leaking memory, it did double the memory footprint while processing a file.